libqtile/backend/x11/drawer.py

```diff
@@ -130,11 +130,7 @@ def _paint(self):
             ctx.set_source_surface(self.surface, 0, 0)
             ctx.paint()
 
-            # If the widget is not being reflected then clear RecordingSurface of operations
-            # If it is, we need to keep the RecordingSurface contents until the mirrors have
-            # been drawn
```

Comment:
You've removed these 3 lines but not the 3 identical lines from `../wayland/drawer.py`. Maybe also remove them from there?